CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ### [TBD] - TBD
 ### Changed
+* Narrowed component scanning to be internal base packages instead of `com.hotels.bdp.circustrain`. See (#95)[https://github.com/HotelsDotCom/circus-train/issues/95]. Note this change is _not_ backwards compatible for any Circus Train extensions that are in the `com.hotels.bdp.circustrain` package - these were in effect being 
```

Comment:
The brackets should be the other way around:
`[#95](https://github.com/HotelsDotCom/circus-train/issues/95)` is [#95](https://github.com/HotelsDotCom/circus-train/issues/95)
but
`(#95)[https://github.com/HotelsDotCom/circus-train/issues/95]` is (#95)[https://github.com/HotelsDotCom/circus-train/issues/95]